src/components/SpaceDetailsPage/SpaceDetailsCard.jsx

```diff
@@ -0,0 +1,362 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import cx from 'classnames';
+import { useHistory } from 'react-router-dom';
+
+import {
+  Button,
+  Card,
+  CardHeader,
+  CardMedia,
+  CardContent,
+  Divider,
+  Typography,
+  IconButton,
+  Link,
+} from '@material-ui/core';
+import StarIcon from '@material-ui/icons/Star';
+import NavigateNextIcon from '@material-ui/icons/NavigateNext';
+import ShareIcon from '@material-ui/icons/Share';
+
+import useMobileDevice from '../../hooks/useMobileDevice';
+import ChipList from '../ChipList';
+
+const styles = () => ({
+  root: {
+    '& .MuiSvgIcon-root': {
+      padding: '6px 6px;',
+    },
+  },
+  starIcon: {
+    paddingBottom: '0 !important',
+  },
+  rating: {
+    display: 'block',
+  },
+  headerAction: {
+    margin: '0 !important',
+  },
+  card: {
+    justifyContent: 'center',
+    alignContent: 'center',
+    padding: '15px',
+    margin: '30px',
+  },
+  cardMedia: {
+    margin: 'auto',
+    height: '300px',
+  },
+  reviewDiv: {
+    width: '100%',
+    display: 'inline-flex',
+    flexFlow: 'row',
+    justifyContent: 'space-between',
+  },
+  featuredReview: {
+    margin: '20px',
+    textAlign: 'left',
+    display: 'inline-flex',
+  },
+  seeAllButton: {
+    display: 'inline-flex',
+    float: 'right',
+    margin: 0,
+  },
+  reviewButton: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    margin: '30px',
+  },
+  // todo: this needs to be a hyperlink
+  location: {
+    display: 'flex',
+    'flex-direction': 'row',
+    marginBottom: 15,
+    color: 'rgba(0, 0, 0, 0.6)',
+  },
+  distance: {
+    textAlign: 'right',
+    flexGrow: 2,
+  },
+  address: {
+    marginRight: 30,
+    flexGrow: 1,
+    textAlign: 'left',
+    color: 'black',
+  },
+  chipWrapper: {
+    'margin-top': '15px',
+  },
+  filter: {
+    margin: '15px 20px 15px 0',
+  },
+  nextButton: {
+    float: 'right',
+    padding: 0,
+  },
+  shareButton: {
+    float: 'right',
+  },
+  subtitles: {
+    fontSize: 12,
+    textTransform: 'uppercase',
+    textAlign: 'left',
+    color: 'grey',
+    marginTop: '10px',
+  },
+  mainInformation: {
+    color: 'black',
+  },
+  hoursOfOperation: {
+    display: 'flex',
+  },
+  url: {
+    textDeocration: 'none',
+  },
+});
+
+const SpaceDetailCard = ({
+  id,
+  name,
+  category,
+  averageRating,
+  imageUrl,
+  address,
+  // distance,
+  hoursOfOperation,
+  filters,
+  phoneNumber,
+  website,
+  classes,
+  yelpUrl,
+  overrideClasses,
+}) => {
+  const [isCopied, setIsCopied] = useState(false);
+  const [isMobileOrTablet] = useMobileDevice();
+
+  const history = useHistory();
+  /* if desktop, copy to clipboard unique url for the space.
+   if mobile, share sheet to text, email, whatever is on your phone or tablet
+  */
+  const handleShare = () => {
+    // eslint-disable-next-line
+    if (isMobileOrTablet && navigator.share) {
```

Comment:
Does this work https://github.com/StephenGrider/ESLint-Rallycoding/issues/7#issuecomment-570694528 instead of having to disable lint all these times?